themes/_core/js/cart.js

```diff
@@ -4,13 +4,13 @@ import prestashop from 'prestashop';
 $(document).ready(() => {
   prestashop.on('cart updated', function (event) {
     var refreshURL = $('.-js-cart').data('refresh-url');
-    var requestData = {};
+    var requestData = {
+      action: 'refresh'
```

Comment:
but i can change it to
`<section id="checkout-cart-summary" class="card card-block -js-cart" data-refresh-url="{url entity='cart' params=['ajax' => true]}">
`